Integrations/Flashpoint/Flashpoint.yml

```diff
@@ -0,0 +1,377 @@
+    - contextPath: URL.Malicious.Description
+      description: Description of malicious url.
+      type: string
+    - contextPath: URL.Malicious.Vendor
+      description: Vendor of malicious url.
+      type: string
+    description: Lookup the "URL" type indicator details
+  - name: file
+    arguments:
+    - name: file
+      required: true
+      description: Enter file. It may sha1, md5, sha256 etc.
+    outputs:
+    - contextPath: DBotScore.Indicator
+      description: The indicator that was tested.
+      type: string
+    - contextPath: DBotScore.Score
+      description: The indicator score.
+      type: number
+    - contextPath: DBotScore.Type
+      description: The indicator type.
+      type: string
+    - contextPath: DBotScore.Vendor
+      description: The vendor used to calculate the score.
+      type: string
+    - contextPath: File.Flashpoint.href
+      description: List of indicators reference.
+      type: unknown
+    - contextPath: File.Malicious.Description
+      description: Description of malicious file.
+      type: string
+    - contextPath: File.Malicious.Vendor
+      description: Vendor of malicious file.
+      type: string
+    - contextPath: File.MD5
+      description: MD5 type file.
+      type: string
+    - contextPath: File.SHA1
+      description: SHA1 type file.
+      type: string
+    - contextPath: File.SHA256
+      description: SHA256 type file.
+      type: string
+    description: Lookup the "File" type indicator details
+  - name: email
+    arguments:
+    - name: email
+      required: true
+      description: Enter valid email
+    outputs:
+    - contextPath: DBotScore.Indicator
+      description: The indicator that was tested.
+      type: string
+    - contextPath: DBotScore.Score
+      description: The indicator score.
+      type: number
+    - contextPath: DBotScore.Type
+      description: The indicator type.
+      type: string
+    - contextPath: DBotScore.Vendor
+      description: The vendor used to calculate the score.
+      type: string
+    - contextPath: Account.Email.Flashpoint.href
+      description: List of email references.
+    - contextPath: Account.Email.Malicious.Description
+      description: Description of Malicious email account.
+      type: string
+    - contextPath: Account.Email.Malicious.Vendor
+      description: Vendor of Malicious email.
+      type: string
+    - contextPath: Account.Email.Name
+      description: Name of indicator.
+      type: string
+    description: Lookup the "Email" type indicator details
+  - name: flashpoint-search-intelligence-reports
+    arguments:
+    - name: report_search
+      required: true
+      description: Search report using keyword or text
+    description: Search for the Intelligence Reports using a keyword
+  - name: flashpoint-get-single-intelligence-report
+    arguments:
+    - name: report_id
+      required: true
+      description: Search report by report fpid
```

Comment:
please add more explanation where user should take/copy this report_id.
You can refer to outputs of other command, if those return the report_id